.github/workflows/publish-docker-image.yml

```diff
@@ -17,6 +16,21 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
+      - name: Import GPG key used for testing
+        id: import-gpg
+        uses: crazy-max/ghaction-import-gpg@v4
+        with:
+          gpg_private_key: ${{ secrets.TEST_GPG_PRIVATE_KEY }}
+          passphrase: ${{ secrets.TEST_PASSPHRASE }}
+          git_user_signingkey: true
+          git_commit_gpgsign: true
+
+      - name: Set GPG key user as the Git global user
+        run: |
+          git config --global user.email "${{ steps.import-gpg.outputs.email }}"
+          git config --global user.name "${{ steps.import-gpg.outputs.name }}"
+          git config --global user.signingkey "${{ steps.import-gpg.outputs.keyid }}"
```

Comment:
hi @yeraydavidrodriguez I think you can remove the step `Set GPG key user as the Git global user` and add the [argument](https://github.com/crazy-max/ghaction-import-gpg#inputs) `git_config_global`

```yml
- name: Import GPG key used for testing
  id: import-gpg
  uses: crazy-max/ghaction-import-gpg@v4
  with:
    gpg_private_key: ${{ secrets.TEST_GPG_PRIVATE_KEY }}
    passphrase: ${{ secrets.TEST_PASSPHRASE }}
    git_config_global: true
    git_user_signingkey: true
    git_commit_gpgsign: true
```

Reply:
Fixed.